helm-charts/iqserver/templates/deployment.yaml

```diff
@@ -52,13 +52,36 @@ spec:
             name: nxiq-pv-log
           - mountPath: /etc/nexus-iq-server
             name: config-volume
+          {{- if  .Values.iq.certificates }}
+          - mountPath: /etc/nexus-iq-server/jks
+            name: nxiq-pv-jks
+          {{- end }}
```

Comment:
if the user provided certificates, then mount the jks truststore into the server.